### PR TITLE
#213 set maximum build count to display in the archives section

### PR DESCRIPTION
**Pull request by prashanth-sams**
# Merge checklist
- [ ] TravisCI tests passed
- [ ] Documentation

### CODE REVIEW

**Comment by prashanth-sams** [comment]
note: will fix the issues mentioned in linter

**Comment by prashanth-sams** [comment]
note: will add unit test for this module

**Review by kozakHolota — CHANGES_REQUESTED** [review]

File: pytest_html_reporter/util.py [hunk 1]
```diff
@@ -78,4 +77,21 @@ def clean_screenshots(path):
 
 
 def custom_title(title):
-    ConfigVars._title = title[:26] + '...' if title.__len__() > 29 else title
\ No newline at end of file
+    ConfigVars._title = title[:26] + '...' if title.__len__() > 29 else title
+
+
+def remove_old_archives(path, archive_count):
```
[the remark this answers is not in the document]
Comment (prashanth-sams) [reply pytest_html_reporter/util.py]: @kozakHolota moved inside the HTMLReporter

File: pytest_html_reporter/html_reporter.py [hunk 2]
```diff
@@ -14,14 +14,16 @@
...
 from html_page.test_row import TestRow
-from pytest_html_reporter.util import suite_highlights, generate_suite_highlights, max_rerun
+from pytest_html_reporter.util import suite_highlights, generate_suite_highlights, max_rerun, remove_old_archives
 from pytest_html_reporter.time_converter import time_converter
 from pytest_html_reporter.const_vars import ConfigVars
 
+
 class HTMLReporter(object):
-    def __init__(self, path, config):
+    def __init__(self, path, archive_count, config):
```
[the remark this answers is not in the document]
Comment (prashanth-sams) [reply pytest_html_reporter/html_reporter.py:24]: > Archive count make as a command line option with some default
- Yeah I already have that snippet in `pytest_addoption` method 
```
group.addoption(
        "--archive-count",
        action="store",
        dest="archive_count",
        default="",
        help="set maximum build count to display in the archives section",
    )
```

> During HTMLReporter init just take the cmd param value and set this as the archive_count attribute value?
- I think I did the same - not sure if I am right. can you please give me the snippet for as a rough idea

[the remark this answers is not in the document]
Comment (prashanth-sams) [reply pytest_html_reporter/html_reporter.py:24]: I actually set empty string as default purposely; cos, we allow archives by default.

**Pull request merged**